website/docs/guides/scala-js.md

```diff
@@ -160,17 +160,11 @@ Scala.js options are supported by using directives in Scala CLI:
 
 For more options, see our [`using` directive section](./reference/directives.md#scala-js-options).
 
-## Supported Scala.js Versions
+## Scala.js Versions
 
-In the future, the Scala CLI will be able to support any version of Scala.js independently of the version of scala-cli. But for now, there are some limitations and it isn't possible to use each version of Scala.js.
+From `0.1.3` the Scala CLI is able to support any version of Scala.js independently of the version of scala-cli. Under the hood it is used Scala.js CLI to linking.
```

Comment:
?
```suggestion
From `0.1.3`, Scala CLI supports any version of Scala.js independently of the version of scala-cli. Under the hood it uses Scala.js CLI to handle linking.
```

Reply:
Are you sure about this, btw? Don't we only support versions that we have scala-js-cli launchers for, so `1.9.0` and `1.10.0` with the [current scala-js-cli](https://github.com/scala-cli/scala-js-cli-native-image/releases/tag/v1.1.1-sc4.1)?

Reply:
Yea, it should work but users have to pass explicitly the latest `scala-js-cli` version using `--js-cli-version` parameter.

I should mention about it, thanks for review.